docs/DeveloperGuide.md

```diff
@@ -17,7 +17,7 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 * [Appendix E : Product Survey](#appendix-e--product-survey)
 
 
-## 1. Setting up
+<h2 name="setting-up">1. Setting up</h2>
```

Comment:
I tried to use markdown links only but those lines are started with number, it seems that then can not be properly linked to.